vmdb/app/controllers/ems_container_controller.rb

```diff
@@ -0,0 +1,23 @@
+class EmsContainerController < ApplicationController
+  include EmsCommon        # common methods for EmsInfra/Cloud/Container controllers
+  include ContainersCommonMixin
+
+  before_filter :check_privileges
```

Comment:
does it mean there is a need to define any special privileges/db entries/user related things, or it is available out of the box at least for the admin user?